label_studio/tasks/models.py

```diff
@@ -1001,6 +1001,50 @@ class Meta:
         db_table = 'prediction'
 
 
+class FailedPrediction(models.Model):
+    """
+    Class for storing failed prediction(s) for a task
+    """
+    message = models.TextField(
+        _('message'),
+        default='',
+        blank=True,
+        null=True,
+        help_text='The message explaining why generating this prediction failed'
+    )
+    error_type = models.CharField(
+        _('error_type'),
+        max_length=512,
+        default=None,
+        null=True,
+        help_text='The type of error that caused prediction to fail'
+    )
+    ml_backend_model = models.ForeignKey(
+        'ml.MLBackend',
+        on_delete=models.SET_NULL,
+        related_name='failed_predictions',
+        null=True,
+        help_text='An ML Backend instance that created the failed prediction.',
+    )
+    model_version = models.TextField(
+        _('model version'),
+        default='',
```

Comment:
same as https://github.com/HumanSignal/label-studio/pull/6189/files#r1717446565
```suggestion
        default=None,
```